src/org/zaproxy/zap/extension/pscanrulesAlpha/ZapAddOn.xml

```diff
@@ -9,6 +9,8 @@
 	<![CDATA[
 		Issue 2538: Strict-Transport-Security Header Scanner additional variants.<br>
 		Issue 2716: Implement a passive check for cookies without SameSite set.<br>
```

Comment:
@kingthorin Just to confirm, by "previous change entries" you mean the line 10 and 11, and not only 11, right ?